LEAF_Nexus/admin/index.php

```diff
@@ -256,6 +256,7 @@
 $memberships = $login->getMembership();
 $t_menu->assign('isAdmin', $memberships['groupID'][1]);
 $main->assign('login', $t_login->fetch('login.tpl'));
+$main->assign('leafSecure', Orgchart\Config::$leafSecure);
```

Comment:
There should be a check here to see if that `$leafSecure` variable is set. If not, it should default to `false`. This will prevent errors in case any configs aren't updated properly.

Reply:
done